### PR TITLE
ACC-1043-ACC-1047 Update FL, TX and UT Sutas

### PR DESCRIPTION
**Pull request by Aidancs**
refs: https://spurwork.atlassian.net/browse/ACC-1043
refs: https://spurwork.atlassian.net/browse/ACC-1047
refs: https://spurwork.atlassian.net/browse/ACC-1062

### CODE REVIEW

File: src/config/config.php [hunk 1]
```diff
@@ -316,18 +324,22 @@
...
-            'utah' => [
-                'unemployment' => env('TAXES_UTAH_UNEMPLOYMENT_TAX_RATE', 0.03),
+            'texas' => [
+                'unemployment' => env('TAXES_TEXAS_UNEMPLOYMENT_TAX_RATE', 0.0031),
             ],
 
-            'washingtondc' => [
-                'unemployment' => env('TAXES_WASHINGTONDC_UNEMPLOYMENT_TAX_RATE', 0.027),
+            'utah' => [
+                'unemployment' => env('TAXES_UTAH_UNEMPLOYMENT_TAX_RATE', 0.011),
```
Comment (stevespur) [inline src/config/config.php:332]: was utah supposed to change? was 0.03 before

**Comment by stevespur** [comment]
we probably want new files instead of replacing the older ones don't we?

**Comment by Aidancs** [comment]
> we probably want new files instead of replacing the older ones don't we?

You mean the deleted tests? When we use the config file to store the rate the old tests with old values become obsolete. It looks like you have update the value to get the tests to pass but is there any value to that? t

File: src/config/config.php (hunk 1 above)
Comment (Aidancs) [reply src/config/config.php:352]: Saw there was a 2020 UT folder and assumed I had updated it like Illinois but I had only update the wage base, so this updates the rate also, now

**Comment by stevespur** [comment]
ah, I missed that is was tests

**Pull request merged**